admin/translations/fr.json

```
@@ -976,15 +976,15 @@
     "candidate": {
       "code": "Évènement",
       "events": {
+        "CandidateCertifiableAndEligibleEvent": "Candidat certifiable et éligible à la double certification",
```

Au vu de la description, proposition d'avoir la mention de `DoubleCertification` dans le nom de l'évènement


```suggestion
        "CandidateDoubleCertificationEligibleEvent": "Candidat certifiable et éligible à la double certification",
```